vignettes/bds_finding.Rmd

```diff
@@ -803,39 +803,73 @@ record and what computations must be done with the new records.
 
 ### Example 1 (Creating a New Record):
 
+For adding new records based on selecting a certain record
```

Comment:
Suggest?
To add a new record based on the selection of a certain criterion (e.g. MIN, MAX)

Reply:
Updated